web/test/pages/onboarding/onboarding-starting.test.tsx

```diff
@@ -132,7 +132,7 @@ describe("onboarding - starting a business", () => {
         page.clickNext();
         expect(screen.getByTestId("step-2")).toBeInTheDocument();
         expect(screen.getAllByText(Config.siteWideErrorMessages.errorRadioButton)[0]).toBeInTheDocument();
-        page.chooseEssentialQuestionRadio(industryId, 0);
+        page.chooseEssentialQuestionRadio(industryId, industryId === "employment-agency" ? 1 : 0);
```

Comment:
pasting same comment here
pebble - why are we changing the logic here? what happens when with the original value. If we need have one off logic, then I think the best think to do here is to separate the tests insteag of using it.each